eqs/Cargo.toml

```diff
@@ -10,6 +10,7 @@ async-std = { version = "1.10.0", features = ["unstable", "attributes", "tokio1"
 atomic_store = { version="0.0.2", registry="translucence" }
 bincode = "1.3.3"
 cap-rust-sandbox = { path = "../contracts/rust" }
+commit = { git = "ssh://git@github.com/EspressoSystems/commit.git", rev = "f48cd52c59755eade0605111826eef3df6abdcf8" }
```

Comment:
Should use `https` url.

Reply:
Oops, yeah, I think this getting added predated that change. I'll fix it.